### DISCUSSION TITLE
请求passwall 2的使用指南，规则管理不会用了

### DISCUSSION THREAD

**Discussion by fengertao** [Openwrt-Passwall/openwrt-passwall2#66] [Q&A]
## 你的功能请求是否与问题有关？ 请描述。(必填)

passwall 2以后，不会用了，请求提供使用指南，如果有视频上传youtube更好。
最主要是”规则管理“不会用。
1. 我希望可以方便的在
* ”翻墙白名单“--只有指定的域名和ip才翻墙
* ”翻墙黑名单“--只有指定的域名和ip才直连
* ”全局翻墙“
等几个模式切换，但找不到切换开关。

2. ”规则管理“里可以修改现在的规则，和增加新的规则。
我理解的规则应该由两个部分组成：A.什么样的流量满足本规则（比如协议，域名，IP等). B.满足本规则的流量是翻墙还是直连。
但规则管理里只有A，没有B，所以不会用了。

## 描述你想要的解决方案 (必填)
请求提供使用指南，如果有视频上传youtube更好

## 描述你考虑过的替代方案
请求提供使用指南，如果有视频上传youtube更好

## 附加上下文
NA

**Comment by chika0801** [comment]
我按个人习惯是这样用的
![162794447-000c7606-4bae-40f0-9c55-0d0070c49a3b](https://user-images.githubusercontent.com/88967758/163006236-01d6afdb-021a-4027-be14-d9d53b05f149.jpg)

讨论
https://github.com/xiaorouji/openwrt-passwall2/issues/49#issuecomment-1094523576

说明，因为用的Xray-core的配置文件规则，需要熟悉那边的配置文件的，就觉得非常友好。要解释也不是几句说的清楚了。

这是分流（DNS查询流程说明）大致可参考下

https://github.com/chika0801/Xray-examples/tree/main/VLESS-TCP-XTLS#dns%E6%9F%A5%E8%AF%A2%E6%B5%81%E7%A8%8B%E8%AF%B4%E6%98%8E

**Reply by blakejia** [reply]
发现个问题。玩英雄联盟手游时china匹配不到服务器，强制走代理。根本玩不了游戏

**Reply by jh0072110** [reply]
> 发现个问题。玩英雄联盟手游时china匹配不到服务器，强制走代理。根本玩不了游戏

lol不应该是走udp的么

**Reply by fengertao** [reply]
> 我按个人习惯是这样用的 ![162794447-000c7606-4bae-40f0-9c55-0d0070c49a3b](https://user-images.githubusercontent.com/88967758/163006236-01d6afdb-021a-4027-be14-d9d53b05f149.jpg)
> 
> 讨论 [#49 (comment)](https://github.com/xiaorouji/openwrt-passwall2/issues/49#issuecomment-1094523576)
> 
> 说明，因为用的Xray-core的配置文件规则，需要熟悉那边的配置文件的，就觉得非常友好。要解释也不是几句说的清楚了。
> 
> 这是分流（DNS查询流程说明）大致可参考下
> 
> https://github.com/chika0801/Xray-examples/tree/main/VLESS-TCP-XTLS#dns%E6%9F%A5%E8%AF%A2%E6%B5%81%E7%A8%8B%E8%AF%B4%E6%98%8E

我的passwall 2的截图如下，没有你截图里的那些下拉框域。😢
![image](https://user-images.githubusercontent.com/896069/163541095-58edc073-15a8-4b1a-a6e0-67a21aa19649.png)

**Reply by blakejia** [reply]
选择分流节点。不要选单一节点

**Comment by zanjie1999** [comment]
如何配置gfw模式？gfw墙的才走

试了几次猜到了，域名写geosite:gfw

**Reply by Ymy214** [reply]
怎么猜的，为什么不能搜一下看看自己需要的是哪一个簇，geoip geositeAPI怎么用

**Comment by wuyuxy2022** [comment]
> ## 你的功能请求是否与问题有关？ 请描述。(必填)
> passwall 2以后，不会用了，请求提供使用指南，如果有视频上传youtube更好。 最主要是”规则管理“不会用。
> 
> 1. 我希望可以方便的在
> 
> * ”翻墙白名单“--只有指定的域名和ip才翻墙
> * ”翻墙黑名单“--只有指定的域名和ip才直连
> * ”全局翻墙“
>   等几个模式切换，但找不到切换开关。
> 
> 2. ”规则管理“里可以修改现在的规则，和增加新的规则。
>    我理解的规则应该由两个部分组成：A.什么样的流量满足本规则（比如协议，域名，IP等). B.满足本规则的流量是翻墙还是直连。
>    但规则管理里只有A，没有B，所以不会用了。
> 
> ## 描述你想要的解决方案 (必填)
> 请求提供使用指南，如果有视频上传youtube更好
> 
> ## 描述你考虑过的替代方案
> 请求提供使用指南，如果有视频上传youtube更好
> 
> ## 附加上下文
> NA
我也碰到这问题，请问你怎么解决的，我现想调局域网内某一台电脑为全局模式，不知道怎么调了
